elixir/apps/web/lib/web/live/resources/components.ex

```diff
@@ -190,7 +185,7 @@ defmodule Web.Resources.Components do
   defp pretty_print_filter(%{protocol: :udp, ports: ports}),
     do: "UDP: #{pretty_print_ports(ports)}"
 
-  defp pretty_print_ports([]), do: "any port"
+  defp pretty_print_ports([]), do: "Allowed"
```

Comment:
Is this correct? This is just to build a human-readable summary of allowed ports, so "allowed" feels off here.

Reply:
Yeah, I changed it because it is a little ambiguous -- does "any port" mean all ports are blocked or allowed? This is how it's shown in the Show page with this change:

`Traffic restriction |  TCP: Allowed`


Instead of --

`Traffic restriction |  TCP: any port`

Reply:
Maybe we should just keep it empty then? So `Traffic restrictions: `TCP: 80-8080, UDP`? Because each of those allowed and this would look weird: `TCP: Allowed, UDP: 22`.